### DISCUSSION TITLE
Change the language of the "No data available" message in the Autocomplete field

### DISCUSSION THREAD

**Discussion by timoshechkin** [vuetifyjs/vuetify#16072] [Q&A]
How can I change the language of the "No data available" message in the Autocomplete field?

![2022-11-11_15-47-19](https://user-images.githubusercontent.com/70754051/201324953-1715fe13-01ca-4302-89cd-dae3f2cc48a4.png)

**Comment by KaelWD** [comment]
https://vuetifyjs.com/en/features/internationalization/#getting-started
https://vuetifyjs.com/en/api/v-autocomplete/#props-no-data-text
